optimization/loaddata.py

```diff
@@ -0,0 +1,20 @@
+import networkx
+import json
+import itertools
+from distance import compute_distance
```

Comment:
Where is `distance` coming from?

Reply:
The conclusion was this had come from other code written by the team. It would be good to commit that either in this or another PR.